modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -129,6 +135,61 @@ public void testConfigurationArrayValues() throws Exception {
 			(String[])properties.get("configUntypedStringArray"));
 	}
 
+	@Test
+	public void testConfigurationDeprecatedFileExtension() throws Exception {
+		String configurationPid = _CONFIGURATION_PID_PREFIX.concat(
+			".testConfigurationDeprecatedFileExtension");
+
+		String content = "testKey=\"testValue\"";
+
+		Path configPathDeprecated = Paths.get(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR,
+			configurationPid.concat(".cfg"));
+
+		_configurationPath = Paths.get(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR,
+			configurationPid.concat(".config"));
+
+		try (CaptureAppender captureAppender =
+				Log4JLoggerTestUtil.configureLog4JLogger(
+					"com.liferay.portal.file.install.internal.configuration." +
+						"ConfigurationFileInstaller",
+					Level.WARN)) {
+
+			Files.write(
+				configPathDeprecated,
+				content.getBytes(Charset.defaultCharset()));
+
+			_configuration = _createConfiguration(configurationPid, content);
+
+			List<LoggingEvent> loggingEvents =
+				captureAppender.getLoggingEvents();
+
+			LoggingEvent loggingEvent = loggingEvents.get(0);
+
+			String actualWarningMessage = (String)loggingEvent.getMessage();
+
+			String expectedWarningMessage1 = "Unable to install .cfg file";
+			String expectedWarningMessage2 = "please use .config file instead";
+
+			Assert.assertTrue(
```

Comment:
What's the point of having two separate asserts here? You have the path, you can just assert a single message